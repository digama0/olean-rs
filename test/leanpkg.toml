[package]
name = "test"
version = "0.1"
lean_version = "3.4.2"
path = "."

[dependencies]
