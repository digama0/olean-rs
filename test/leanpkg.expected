init.core
init.logic
init.data.prod
init.data.nat.basic
init.wf
init.data.sigma.basic
init.propext
init.data.setoid
init.data.quot
init.funext
init.function
init.data.bool.basic
init.data.list.basic
init.data.char.basic
init.data.string.basic
init.data.subtype.basic
init.data.fin.basic
init.data.unsigned.basic
init.data.sum.basic
init.data.nat.div
init.data.repr
init.data.ordering.basic
init.coe
init.data.to_string
init.meta.name
init.category.functor
init.category.applicative
init.category.alternative
init.category.monad
init.category.lift
init.data.option.basic
init.meta.options
init.meta.format
init.util
init.category.combinators
init.category.monad_fail
init.meta.exceptional
init.meta.level
init.meta.rb_map
init.meta.expr
init.meta.task
init.meta.declaration
init.meta.environment
init.meta.pexpr
init.meta.interaction_monad
init.meta.tactic
init.meta.relation_tactics
init.meta.contradiction_tactic
init.meta.constructor_tactic
init.meta.injection_tactic
init.meta.occurrences
init.meta.rewrite_tactic
init.meta.has_reflect
init.meta.lean.parser
init.meta.attribute
init.meta.simp_tactic
init.meta.interactive_base
init.meta.set_get_option_tactics
init.meta.smt.congruence_closure
init.meta.rec_util
init.meta.mk_has_reflect_instance
init.meta.mk_has_sizeof_instance
init.meta.mk_inhabited_instance
init.meta.derive
init.meta.match_tactic
init.meta.congr_lemma
init.meta.congr_tactic
init.meta.interactive
init.category.id
init.category.except
init.category.state
init.category.reader
init.category.option
init.category.lawful
init.category.default
init.data.punit
init.data.set
init.data.basic
init.version
init.classical
init.cc_lemmas
init.meta.fun_info
init.meta.ac_tactics
init.meta.backward
init.meta.mk_dec_eq_instance
init.meta.converter.conv
init.meta.converter.interactive
init.meta.converter.default
init.meta.vm
init.meta.comp_value_tactics
init.meta.smt.ematch
init.category.default
init.meta.smt.smt_tactic
init.meta.smt.rsimp
init.meta.smt.interactive
init.meta.smt.default
init.meta.async_tactic
init.meta.ref
init.meta.hole_command
init.meta.default
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.data.sigma.lex
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.algebra.classes
init.algebra.order
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.meta.decl_cmds
init.algebra.group
init.algebra.ordered_group
init.algebra.ring
init.algebra.ordered_ring
init.algebra.field
init.algebra.ordered_field
init.algebra.functions
init.data.nat.lemmas
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.data.list.lemmas
init.data.list.instances
init.data.list.qsort
init.data.string.instances
init.meta.well_founded_tactics
init.algebra.norm_num
init.algebra.default
init.data.sigma.default
init.data.nat.bitwise
init.data.nat.default
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.data.char.lemmas
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.data.nat.gcd
init.data.int.basic
init.data.int.order
init.data.int.comp_lemmas
init.data.int.bitwise
init.data.int.default
init.data.char.classes
init.data.char.default
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.data.bool.lemmas
init.data.string.ops
init.data.string.default
init.data.list.default
init.data.sum.default
init.data.subtype.instances
init.data.subtype.default
init.data.int.default
init.data.nat.default
init.data.bool.default
init.data.bool.default
init.ite_simp
init.data.array.basic
init.data.nat.default
init.data.array.slice
init.data.array.default
init.data.bool.default
init.data.nat.default
init.data.fin.ops
init.data.fin.default
init.data.unsigned.ops
init.data.unsigned.default
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.data.ordering.lemmas
init.data.ordering.default
init.data.ordering.default
init.data.rbtree.basic
init.data.rbtree.default
init.data.rbtree.default
init.data.rbmap.basic
init.data.rbmap.default
init.data.option.instances
init.data.default
init.default
init.category.default
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.algebra.default
init.data.sigma.default
init.data.nat.default
init.data.char.default
init.data.string.default
init.data.list.default
init.data.sum.default
init.data.subtype.default
init.data.int.default
init.data.array.default
init.data.bool.default
init.data.fin.default
init.data.unsigned.default
init.data.ordering.default
init.data.rbtree.default
init.data.rbtree.default
init.data.rbmap.default
init.data.default
init.default
init.category.default
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.algebra.default
init.data.sigma.default
init.data.nat.default
init.data.char.default
init.data.string.default
init.data.list.default
init.data.sum.default
init.data.subtype.default
init.data.int.default
init.data.array.default
init.data.bool.default
init.data.fin.default
init.data.unsigned.default
init.data.ordering.default
init.data.rbtree.default
init.data.rbtree.default
init.data.rbmap.default
init.data.default
init.default
init.category.default
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.algebra.default
init.data.sigma.default
init.data.nat.default
init.data.char.default
init.data.string.default
init.data.list.default
init.data.sum.default
init.data.subtype.default
init.data.int.default
init.data.array.default
init.data.bool.default
init.data.fin.default
init.data.unsigned.default
init.data.ordering.default
init.data.rbtree.default
init.data.rbtree.default
init.data.rbmap.default
init.data.default
init.default
data.buffer
init.category.default
init.meta.converter.default
init.meta.smt.default
init.meta.default
init.algebra.default
init.data.sigma.default
init.data.nat.default
init.data.char.default
init.data.string.default
init.data.list.default
init.data.sum.default
init.data.subtype.default
init.data.int.default
init.data.array.default
init.data.bool.default
init.data.fin.default
init.data.unsigned.default
init.data.ordering.default
init.data.rbtree.default
init.data.rbtree.default
init.data.rbmap.default
init.data.default
init.default
system.random
system.io_interface
system.io
simple
